apis/nucleus/src/components/Cell.jsx

```diff
@@ -286,9 +286,13 @@ const Cell = forwardRef(
   ({ halo, model, initialSnOptions, initialSnPlugins, initialError, onMount, currentId }, ref) => {
     const { app, types } = halo;
     const { nebbie } = halo.public;
-    const { disableCellPadding = false } = halo.context || {};
-
-    const { theme: themeName, translator, language, keyboardNavigation } = useContext(InstanceContext);
+    const {
+      theme: themeName,
+      translator,
+      language,
+      keyboardNavigation,
+      disableCellPadding = false,
```

Comment:
I moved `disableCellPadding` into this statement instead of the previous check on halo.context.